ai/sources/_liquid-rules.liquid

```diff
@@ -0,0 +1,51 @@
+valid_filters = [
+  {% assign categories = filters | map: 'category' | uniq | sort -%}
+
+  {% for category in categories %}
+    // {{ category }}
+    {% assign category_filters = filters | where: 'category', category -%}
+    {%- for filter in category_filters -%}
+      { name: "{{ filter.name }}", usage: "{{ filter.syntax }}" },
+    {% endfor -%}
+  {% endfor %}
+]
+
+valid_tags = [
+  {%- for tag in tags %}
+    "{{ tag.name }}",
+  {%- endfor %}
+]
+
+valid_objects = [
+  {%- for object in objects %}
+    {%- if object.access.global %}
+    "{{ object.name }}",
+    {%- endif -%}
+  {%- endfor %}
+]
+
+validation_rules = {
+  syntax: {
+    - Use {% liquid %} for multiline code
+    - Use {% # comments %} for inline comments
```

Comment:
Do these things get removed by liquidjs? :p, should we `{% raw %}` them?

Reply:
Excellent catch!! 🙇 [Fixed](https://github.com/Shopify/theme-liquid-docs/pull/968/commits/48b5ac07d70106143b63aa96e83dab3d45b56736)